package.json

```diff
@@ -5,15 +5,29 @@
   "dependencies": {
     "@material-ui/core": "^3.5.1",
     "@material-ui/icons": "^3.0.1",
+    "connected-react-router": "^6.0.0",
+    "cryptocurrency-icons": "^0.9.3",
+    "dotenv": "^6.2.0",
```

Comment:
We don't use configuration at build time
you need to commit a file in public/env.js
the application is build using docker and we mount env.js at runtime after image is built.
please see my example code: https://github.com/openware/swagger/blob/master/public/env.js
https://github.com/openware/swagger/blob/master/public/index.html#L29

Reply:
remove dotenv